### PR TITLE
chore: update bot author info and add dup notes for release notes

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: scripts/release_notes_update_pr_author_info_add_dup.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+def update_pr_author_and_release_notes(excel_path):
+
+    # Open the excel file
+    workbook = openpyxl.load_workbook(excel_path)
+    # Specify the target sheet
+    sheet = workbook['pr_for_release_note']
+
+    # Get the sheet header
+    header = next(sheet.iter_rows(min_row=1, max_row=1, values_only=True)) #限定只读取第一行的数据
```
Comment (qiancai) [inline scripts/release_notes_update_pr_author_info_add_dup.py:92]:
```suggestion
    header = next(sheet.iter_rows(min_row=1, max_row=1, values_only=True)) #readind the first line in the excel
```

File: scripts/release_notes_update_pr_author_info_add_dup.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    pr_author_index = header.index('pr_author')
+    pr_link_index = header.index('pr_link')
+    pr_title_index = header.index('pr_title')
+    pr_formated_rn_index = header.index('formated_release_note')
+    pr_last_col_index = sheet.max_column
+    sheet.insert_cols(pr_last_col_index + 1) # Insert a new column for the dup release notes
+    sheet.cell(row=1, column=pr_last_col_index + 1, value='published_release_notes') # Set a column name
+    # 遍历每一行
+    for row_index, row in enumerate(sheet.iter_rows(min_row=2, values_only=True), start=2):
+        # 如果 pr_author 列的值为 ti-chi-bot 或 ti-srebot
```
Comment (qiancai) [inline scripts/release_notes_update_pr_author_info_add_dup.py:104]:
```suggestion
    # Go through each row
    for row_index, row in enumerate(sheet.iter_rows(min_row=2, values_only=True), start=2):
        # If pr_author is ti-chi-bot or ti-srebot
```

File: scripts/release_notes_update_pr_author_info_add_dup.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
+# This script can replace the bot author info in the release note table with the actual PR authors and add the history duplicated release notes based on issue links and author info.
+# Before running this script, you need to get a GitHub personal access token and save it in a text file.
```
Comment (qiancai) [inline scripts/release_notes_update_pr_author_info_add_dup.py:2]:
```suggestion
# Before running this script, you need to get a GitHub personal access token (https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and save it in a text file.
```

File: scripts/release_notes_update_pr_author_info_add_dup.py [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+from github import Github
+import re
+import openpyxl
+import os
+
+release_note_excel = r'/Users/userid/Downloads/patch_6.5.1_release_note_test copy.xlsx'
+ext_path = r'/Users/userid/Documents/GitHub/githubid/docs-cn/releases'  # The path of the existing release notes
+main_path = r'/Users/githubid/Documents/GitHub/githubid/docs-cn/releases/release-6.5.1.md'  # The path of the release notes in preparation
+with open("/Users/githubid/Documents/gh_token.txt", "r") as f: # Read the GitHub personal access token from the token.txt file
```
Comment (qiancai) [inline scripts/release_notes_update_pr_author_info_add_dup.py:12]:
```suggestion
with open("/Users/githubid/Documents/gh_token.txt", "r") as f: # Read the GitHub personal access token from the gh_token.txt file
```

**Review by ran-huang — APPROVED** [review]
LGTM. Can we merge this now?

**Pull request merged**